src/main/java/com/google/api/codegen/PageStreamingConfig.java

```diff
@@ -26,6 +26,7 @@
  * PageStreamingConfig represents the page streaming configuration for a method.
  */
 public class PageStreamingConfig {
+  private final Field requestPageSizeField;
```

Comment:
You can just call this `pageSizeField`, since it's not ambiguous, unlike `token`